scripts/linters/general_purpose_linter.py

```diff
@@ -215,7 +215,9 @@
         'message': 'Please do not use innerHTML property.',
         'excluded_files': (
             'core/templates/Polyfills.ts',
-            'core/templates/filters/translate.pipe.spec.ts'),
+            'core/templates/filters/translate.pipe.spec.ts',
+            'core/templates/components/ck-editor-helpers/' +
+            'ck-editor-copy-content-service.spec.ts'),
```

Comment:
I don't see any `r'innerHTML` inside spec file, why is this added here?

Reply:
It is on line 31 on that file:

const generateContent = (html: string): HTMLElement => {
  const container = document.createElement('template');
  container.**innerHTML** = `<angular-html-bind>${html.trim()}</angular-html-bind>`;
  return <HTMLElement>(container.content.firstChild.firstChild);
};